pkg/defaults/mesh/mesh.go

```diff
@@ -87,3 +90,17 @@ func EnsureDefaultMeshResources(resManager manager.ResourceManager, meshName str
 	}
 	return nil
 }
+
+func ensureDefaultResource(resManager manager.ResourceManager, res model.Resource, resourceKey model.ResourceKey) (err error, created bool) {
+	err = resManager.Get(context.Background(), res, store.GetBy(resourceKey))
```

Comment:
While we're touch this code can we fix the context and pass it down to `EnsureDefaultResources` instead of using `context.Background`?

Reply:
sure